.github/workflows/pull-request-commentor.yaml

```
@@ -0,0 +1,93 @@
+---
+name: Add comment
+# yamllint disable-line rule:truthy
+on:
+  pull_request_target:
+    branches:
+      - devel
+      - "release-v*"
+    types:
+      - labeled
+jobs:
+  add-comment:
+    if: github.event.label.name == 'ok-to-test'
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Add comment to trigger external storage tests for Kubernetes 1.22
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            /test ci/centos/k8s-e2e-external-storage/1.22
```

```suggestion
            /test all
```
Can we instead just use `all` instead of multiple comments  ?

---

We dont want to trigger all (some experimental) tests we want to trigger individual tests.

---

okay